deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/ago/pvc.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: data
+spec:
+  resources:
+    requests:
+      storage: 10Ti
```

This PVC is only going to store `indexer.Value` objects whishould be very small compared to all the multihashes. 

Let's reduce this and expand as needed.
```suggestion
      storage: 1Ti
```